core-web/libs/dotcms-webcomponents/src/elements/dot-contentlet-thumbnail/dot-contentlet-thumbnail.tsx

```diff
@@ -66,8 +69,10 @@ export class DotContentletThumbnail {
                     />
                 ) : this.renderImage ? (
                     <div
-                        class={`thumbnail ${imgClass}`}
-                        style={{ 'background-image': backgroundImageURL }}>
+                        class={`thumbnail ${imgClass}${svgClass}`}
+                        style={{
+                            'background-image': backgroundImageURL
+                        }}>
```

Comment:
What happens when you use an `SVG` Image and the `backgroundImage` input/props is set to `true`?

Reply:
It works as expected with and without the prop, here an example with the prop activate (Content Search) and without the prop (Binary Field with contentlet):

https://github.com/user-attachments/assets/96ea89b1-5a15-4ca3-9afd-76d2e89e5d77